Gems/AWSCore/Code/Include/Framework/ServiceRequestJob.h

```diff
@@ -241,6 +218,33 @@ namespace AWSCore
         /// for replacing these parts of the url.
         const Aws::String& m_requestUrl;
 
+        /// Constructor for creating Jobs that can handle queued responses
```

Comment:
Nit: May worth specifying the job type (ServiceRequestJob) in the description.

Reply:
done